### PR TITLE
completed 4

### PR DESCRIPTION
**Pull request by kontsio1**

### CODE REVIEW

File: src/apis.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+const gamesApi = axios.create({baseURL: 'https://games-selector.onrender.com/api'})
+
+export const getReviews = () => {
+    return gamesApi.get('/reviews').then((res) => {
+        console.log(res.data.reviews)
```
Comment (leekli) [inline src/apis.js:7]: Let's get rid of this

**Comment by kontsio1** [comment]
I added review_id={review_id} as a param passed down to the ReviewCard component in the ReviewsList component for the next task

**Review by leekli — CHANGES_REQUESTED** [review]
Hey Konstantinos, well done so far! A couple of changes / suggestions on this ticket:

* The homepage isn't immediately obvious to me that I should click it to see reviews, have a look at the design of this, either have a set home page with a clear 'reviews' button/link or just set the home page (/) to go straight to ReviewsList maybe?
* There's a console log to remove
* There isn't an indication to me that the reviews are loading when it's doing the api calls - please could you have a look at, and implement a loading design pattern.
* On line 7 in ReviewCard, in the  tag, have a think about a suitable alt tag to add to this, for accessibility. 
* For loading all the reviews... you may want to render these as an unordered list, it doesn't need to be a list however if you don't do an unordered list then you'll need to look at the div's which you have in ReviewCard.jsx and have a more appropriate semantic HTML tag, however this may change your styling....

Have a look at the above, implement the requested changes and pop us another pull request in when ready :)

File: src/components/ReviewCard.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom"
+
+ 
+ export const ReviewCard = ({review_id, title, owner, review_img_url, votes}) => {
+    return (
+        <Link to={`/reviews/${review_id}`}>
+            <div>
```
Comment (lewispricey) [inline src/components/ReviewCard.jsx:7]: As you're now using an ordered list, these divs may be more semantic as list items

**Review by lewispricey — APPROVED** [review]
Great work Konstantinos, thanks for getting Lee's changes made!

I've just left a small comment on the ReviewCard divs but other than that great work.

I can see commented out code for your next endpoint, try to remember to move branch when you start working on other tickets so keep everything together.

Feel free to merge once you've looked at those divs 🚀

**Pull request merged**